stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_giphy.xml

```diff
@@ -14,12 +14,11 @@
         app:layout_constraintTop_toTopOf="parent"
         />
 
-    <com.google.android.material.card.MaterialCardView
```

Comment:
What was causing the crash exactly? `MaterialCardView` is a library class, it should work on all API versions, shouldn't it?

And if we do really have to remove it, we should also check the other two places in the project where we're using it.

Reply:
It doesn't work with the background option somewhy. Exception happened in LayoutInflater class. MaterialCardView works on all API versions, except in combination with that attribute. I couldn't find any similar issue on Github with this problem

Reply:
Gonna look into this a bit